geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/records/SimpleFeatureRecordConverter.scala

```diff
@@ -259,6 +282,12 @@ object SimpleFeatureRecordConverter extends LazyLogging {
       case ObjectType.BOOLEAN  => new BooleanFieldConverter(name)
       case ObjectType.DATE     => new DateFieldConverter(name)
       case ObjectType.UUID     => new UuidFieldConverter(name)
+//<<<<<<< HEAD
+//      case ObjectType.GEOMETRY => GeometryToRecordField(name, encoding, encodings)
```

Comment:
In my first version of updating this function, I ended up adding a case to handle ObjectType.GEOMETRY.  Is that still useful / necessary?

Reply:
i added the geometry checks outside this method since they require some other config (geom encoding, etc), i.e. [here](https://github.com/geomesa/geomesa-nifi/blob/84f08e2d9efe93030488894c461a7574ddb1bacf/geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/records/SimpleFeatureRecordConverter.scala#L160-L169) and [here](https://github.com/geomesa/geomesa-nifi/blob/84f08e2d9efe93030488894c461a7574ddb1bacf/geomesa-datastore-bundle/geomesa-datastore-processors/src/main/scala/org/geomesa/nifi/datastore/processor/records/SimpleFeatureRecordConverter.scala#L225-L237)
I'm open to other approaches, though.

Reply:
Meh, in my first approach, I had widened one of the methods: https://github.com/geomesa/geomesa-nifi/pull/83/files#diff-694182df46af318ff33b4b7c727ff701R125 to keep the match working the same way.  

I think it is 6 of one and half of dozen of the other.  (Meaning that we can leave it the way that is.)  

If you think my first PR's approach is better, lemme know and I can refactor it in.